MapboxVisionTests/RecordingQuotaTests.swift

```diff
@@ -0,0 +1,124 @@
+import XCTest
+@testable import MapboxVision
+
+typealias Byte = Int64
+
+private let bytesInKByte: Byte = 1024
+private let kByteInMByte: Byte = 1024
+
+private let kByte = bytesInKByte
+private let mByte = kByteInMByte * kByte
+
+private let secondsInMinute: TimeInterval = 60
+private let minutesInHour: TimeInterval = 60
+
+private let minute = secondsInMinute
+private let hour = secondsInMinute * minute
+
+private let memoryLimit = 300.0 // mb
+private let networkingMemoryLimit = 30 * mByte
+private let updatingInterval = 1 * hour
+
+class RecordingQuotaTests: XCTestCase {
+    // MARK: - Properties
+
+    private var refreshInterval: TimeInterval!
+    private var initialMemoryQuota: Byte!
+    private var recordingQuota: RecordingQuota!
+
+    // MARK: - Test functions
+
+    override func setUp() {
+        super.setUp()
+        refreshInterval = TimeInterval.random(in: 5...10)
+        initialMemoryQuota = Byte.random(in: 5...10) * kByte
```

Comment:
The problem with this code that test-cases that use this values aren't reproducible. If some tests fail we won't know particular which was used. Moreover potential bug may not be uncovered until after a number of runs of the test.
WDYT about using specific values?

Reply:
Let's avoid random values in that particular case.